pallets/loans/src/entities/pricing/external.rs

```diff
@@ -16,15 +16,25 @@ use sp_runtime::{
 
 use crate::pallet::{Config, Error, PoolIdOf, PriceOf};
 
+/// Define the max borrow amount of a loan
+#[derive(Encode, Decode, Clone, PartialEq, Eq, TypeInfo, RuntimeDebug, MaxEncodedLen)]
+pub enum MaxBorrowAmount<Balance> {
```

Comment:
Maybe we should rename this to `ExternalMaxBorrowAmount` and vice verse for the internal variant, since otherwise the metadata will have two enums with the same name?

Reply:
I think `TypeInfo` should somehow add the path information of where this type is, otherwise there would be a lot of collisions with any other pallet type in Substrate.

Let me check about this

Reply:
Checked it uses `core::any::TypeId` which is the same one I've used for differentiate calls in `mock-builder` and it adds the path of the type. So both types are differents.

Reply:
Alright 👍 Thanks for checking